x-pack/plugins/cases/public/containers/user_profiles/use_get_current_user_profile.test.ts

```diff
@@ -36,11 +36,13 @@ describe.skip('useGetCurrentUserProfile', () => {
   it('calls getCurrentUserProfile with correct arguments', async () => {
     const spyOnGetCurrentUserProfile = jest.spyOn(api, 'getCurrentUserProfile');
 
-    const { result, waitFor } = renderHook(() => useGetCurrentUserProfile(), {
+    const { waitFor } = renderHook(() => useGetCurrentUserProfile(), {
       wrapper: appMockRender.AppWrapper,
     });
 
-    await waitFor(() => result.current.isSuccess);
+    await waitFor(() => {
+      expect(spyOnGetCurrentUserProfile).toBeCalled();
+    });
```

Comment:
Same as above(removing line 47)

```suggestion
    await waitFor(() => {
      expect(spyOnGetCurrentUserProfile).toBeCalledWith({
        security: expect.anything(),
      });
    });
```